ruby/test/vaccine_demo.rb

```diff
@@ -1,80 +1,131 @@
+# frozen_string_literal: true
+
 require 'trinsic_services'
 require 'services/service_base'
 require 'services/account_service'
+require 'services/provider_service'
 require 'services/credential_service'
+require 'services/credential_template_service'
 require 'services/wallet_service'
 require 'json'
 
-def data_base_path
-  File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "devops", "testdata"))
+def do_template(template_service)
+  # createTemplate() {
+  request = Trinsic::Template_V1::CreateCredentialTemplateRequest.new(name: 'VaccinationCertificate',
+                                                                      allow_additional_fields: false)
+  request.fields['firstName'] = Trinsic::Template_V1::TemplateField.new(description: 'Given name')
```

Comment:
```suggestion
  request.fields['firstName'] = Trinsic::Template_V1::TemplateField.new(description: 'First name of vaccine recipient')
```

To ensure template parity across languages